README.md

```diff
@@ -1,5 +1,8 @@
 ## Identity Verification with Amazon Rekognition
 
+> [!WARNING]
+> This repository is archived. Try [AWS Samples - Rekognition Identity Verification](https://github.com/aws-samples/rekognition-identity-verification) instead.
```

Comment:
```suggestion
> This repository is archived and no longer maintained. It may no longer work, or refer to third-party libraries with known problems. We suggest [AWS Samples - Rekognition Identity Verification](https://github.com/aws-samples/rekognition-identity-verification) as an alternative to this sample.
```

Reply:
Good suggestion. Thank you Chris. I have incorporated it.